social-network/src/main/java/com/levi9/socialnetwork/Service/impl/PostServiceImpl.java

```diff
@@ -76,6 +76,23 @@ public List<Post> getAllPostsFromFriends(Long userId) throws ResourceNotFoundExc
     }
     
 
+    
+    public List<Post> getAllPostsFromFriends(Long userId) throws ResourceNotFoundException
+    {
```

Comment:
Move { into the previous line. Same for methods bellow.

Reply:
Refactoring completed